VideoLocker/res/values/styles.xml

```diff
@@ -22,8 +22,10 @@
         <item name="popupSpinnerStyle">@style/edX.Widget.Spinner.Popup</item>
         <item name="drawerArrowStyle">@style/DrawerArrowStyle</item>
         <item name="disableableButtonTextColor">@color/disableable_button_text</item>
-        <item name="android:listChoiceBackgroundIndicator">@drawable/list_item_selector</item>
-        <item name="listChoiceBackgroundIndicator">@drawable/list_item_selector</item>
+        <item name="android:listChoiceBackgroundIndicator">@drawable/view_selector</item>
+        <item name="listChoiceBackgroundIndicator">@drawable/view_selector</item>
```

Comment:
Maybe make these reference the `selectableItemBackground` theme attribute directly:

```` java
<item name="android:listChoiceBackgroundIndicator">?android:attr/selectableItemBackground</item>
<item name="listChoiceBackgroundIndicator">?android:attr/selectableItemBackground</item>
````

It's up to you though.